### PR TITLE
chore: update crate dependencies

### PR DESCRIPTION
**Pull request by sevenautumns**

### CODE REVIEW

File: partition/src/lib.rs [hunk 1]
```diff
@@ -107,7 +107,7 @@ pub(crate) static SIGNAL_STACK: Lazy<MmapMut> = Lazy::new(|| {
         .unwrap()
 });
 
-pub(crate) static SYSCALL: Lazy<RawFd> = Lazy::new(|| {
+pub(crate) static SYSCALL: Lazy<OwnedFd> = Lazy::new(|| {
```
Comment (dadada) [inline partition/src/lib.rs:110]: Can we do this in a separate (logical) commit or is it required for the update?

**Review by dadada — CHANGES_REQUESTED** [review]
See comments above

File: partition/src/lib.rs (hunk 1 above)
Comment (sevenautumns) [reply partition/src/lib.rs]: yes. done

File: hypervisor/src/lib.rs [hunk 2]
```diff
@@ -69,6 +69,7 @@ pub fn run_hypervisor() -> LeveledResult<()> {
             .cgroups()
             .expect("unable to retrieve my parent cgroup");
         let cgroup_path = cgroups
+            .0
```
Comment (dadada) [inline hypervisor/src/lib.rs:72]: .into_iter()

**Review by dadada — APPROVED** [review]

**Pull request merged**